CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+
+## Maintainers
+
+To keep every PR have been checked, we make a team maintainers. Any PR(include owners' PR) MUST be reviewed and by other two maintainers to check before merged.
+Maintainers should be a contributor of gitea(or gogs) and contributed more than 4 PRs(included). And a contributar should apply as a maintainer in [gitter gitea develop](https://gitter.im/go-gitea/develop).
+And the owners or the maintainers team maintainer could invite the contributor. A maintainer should spend some time on code view PRs. If some maintainer have no time
+to do that, he should apply to leave maintainers team and we will give him an honor to be as a member of advisor team. Of course, if an advisor have time to code view, welcome it back to maintainers team.
+If some one have no time to code view and forget to leave the maintainers, the owners have the power to move him from maintainers team to advisors team.
+
+## Owners
+
+Since gitea is a pure community organization with no any company support now, to keep it development healthly We will elect the owners every year. Every time we will elect three owners.
+All the contributers could vote for three owners, one is the main owner, the other two are assistant owners. When the new owners have been elected, the old owners MUST move the power to the new owners. 
+If someone owners don't obey this CONTRIBUTING, all the contributors could fork a new project and continue the project. 
+
+After the election, the new owners should say he agree with the CONTRIBUTING on the [Gitter Gitea Channel](https://gitter.im/go-gitea/gitea). Below is the word to speak
+
+```
+I'm glad to be as an owner of gitea, I agree with [CONTRIBUTING](CONTRIBUTING.md). I will spend part of my time on gitea and lead the development of gitea.
+```
+
+For a honor to the owners, this document will add the history owners below:
+
+2016 - 2017 lunny<xiaolunwen@gmail.com> tboerger<thomas@webhippie.de> bkcsoft<kim.carlbacker@gmail.com>
+
+## Versions
+
+Gitea has one master as a tip branch and have many version branch such as v0.9. v0.9 is a release branch and we will tag v0.9.0 both for binary download.
+If v0.9.0 have some bugs, we will accept PR on v0.9 and publish v0.9.1 and merge bug PR to master.
+
+Branch master is a tip version, so if you wish a production usage, please download the latest release tag version. All the branch will be protected via github,
+All the PRs to all the branches should be review by two maintainers and pass the automatic tests.
+
+## Copyright
+
+Code that you contribute should use the standard copyright header:
+
+```
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+```
+
+Files in the repository are copyright the year they are added. It is not
+necessary to update the copyright year on files that you change.
```

Comment:
I actually think it is needed to add an year of last change as in Copyright (C) 2014-2016. And use multiple lines for multiple copyright owners (Gogs and Gitea contributors..)